pkg/interceptors/cel/cel_test.go

```diff
@@ -0,0 +1,200 @@
+package cel
+
+import (
+	"bytes"
+	"io/ioutil"
+	"net/http"
+	"reflect"
+	"testing"
+
+	"github.com/google/cel-go/common/types"
+	"github.com/google/cel-go/common/types/ref"
+	"github.com/tektoncd/pipeline/pkg/logging"
+	triggersv1 "github.com/tektoncd/triggers/pkg/apis/triggers/v1alpha1"
+	fakekubeclient "knative.dev/pkg/client/injection/kube/client/fake"
+	rtesting "knative.dev/pkg/reconciler/testing"
+)
+
+// Allow configuration via a config map
+
+func TestInterceptor_ExecuteTrigger(t *testing.T) {
+	type args struct {
+		payload []byte
+	}
+	tests := []struct {
+		name    string
+		CEL     *triggersv1.CELInterceptor
+		args    args
+		want    []byte
+		wantErr bool
+	}{
+		{
+			name: "simple body check with matching body",
+			CEL: &triggersv1.CELInterceptor{
+				Filter: "body.value == 'testing'",
+			},
+			args: args{
+				payload: []byte(`{"value":"testing"}`),
+			},
+			want:    []byte(`{"value":"testing"}`),
+			wantErr: false,
```

Comment:
Mixing both error and non-error tests makes it a little harder to follow which cases should and should not work. Can we split these cases into 2 separate tests?